app/router.go

```diff
@@ -129,7 +131,8 @@ var topologyRegistry = map[string]topologyView{
 }
 
 type topologyView struct {
-	human    string
-	parent   string
-	renderer render.Renderer
+	human       string
+	parent      string
+	renderer    render.Renderer
+	isMultiHost bool // does the view involve information from multiple hosts?
```

Comment:
Sure, I can do that, I avoided it to save a pass on `RendarableNode.Origins` but I guess that won't buy us much.